mne/viz/evoked.py

```diff
@@ -1463,15 +1614,14 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
         area. All instances must have the same shape - channel numbers, time
         points etc.
         If dict, keys must be of type str.
-    picks : int | list of int
+    picks : None | int | list of int
         If int or list of int, the indices of the sensors to average and plot.
-        Must all be of the same channel type.
-        If the selected channels are gradiometers, the corresponding pairs
-        will be selected.
         If multiple channel types are selected, one figure will be returned for
         each channel type.
-        If an empty list, `gfp` will be set to True, and the Global Field
-        Power plotted.
+        If the selected channels are gradiometers, the signal coming from
+        gradiometer corresponding pairs with be combined.
+        If None, it defaults to all data channels, in which case the global
+        field power will be plotted for all channel type available.
     gfp : bool
```

Comment:
If the selected channels are gradiometers, the signal from corresponding (gradiometer) pairs will be combined.